types/src/delayed_fields.rs

```diff
@@ -87,6 +87,18 @@ impl From<(u32, u32)> for DelayedFieldID {
     }
 }
 
+impl From<DelayedFieldID> for SizedID {
+    fn from(id: DelayedFieldID) -> Self {
+        Self::new(id.unique_index, id.width)
```

Comment:
Now, I'm confused again why we need to have `DelayedFieldID` and `SizedID` separately. They store exactly the same info -- unique_index and width.

Reply:
Just saw the slack messages. So, we are renaming SizedID to DelayedFieldID?

Reply:
Not even renaming, replaced it with DelayedFieldID to avoid confusion!